src/daml/Daml/Finance/App/Quoting/Model.daml

```diff
@@ -0,0 +1,41 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.App.Quoting.Model where
+
+import Daml.Finance.Interface.Types.Common.Types (Id, InstrumentQuantity)
+
+data Side
+    = Buy
+    | Sell
+  deriving (Eq, Show)
+
+template Quote
```

Comment:
It feels very similar to the `Order` used in the `Trading` service.

Is the difference that a `Quote` is non binding?

Reply:
Got it, `Quote` comes from the provider whereas order comes from `Customer`

Reply:
Yeah, this could probably be unified. It was a quick-and-dirty implementation